assets/app/views/project.html

```diff
@@ -2,7 +2,7 @@
   <project-page>
     <div ng-controller="OverviewController">
       <tasks></tasks>
-      <h1>Project {{project.displayName || project.metadata.name}}</h1>
+      <h1>Project {{project | displayName || project.metadata.name}}</h1>
```

Comment:
I get the following error when I go to the web console.

Open a browser and go to ```https://localhost:8443/console```

Expected results:
See the correct page

Actual results:
See a messed up page, and console shows following error
Token '||' is an unexpected token at column 23 of the expression [project | displayName || project.metadata.name] starting at [||